pyanaconda/ntp.py

```diff
@@ -107,25 +105,14 @@ def ntp_server_working(server_hostname, nts_enabled):
     :return: True if the given server is reachable and working, False otherwise
     :rtype: bool
     """
-    try:
-        # ntplib doesn't support NTS
-        if nts_enabled:
-            s = socket.create_connection((server_hostname, 4460), 2)
```

Comment:
@mlichvar, can I replace your solution for nts with the one below? It seems to work fine.

Reply:
Yes, I think that's a good idea. Thanks.

Reply:
NETWORK_CONNECTION_TIMEOUT seems to be 46 seconds. Isn't that a bit too long for this check? I'd suggest something like 8 seconds, or so.

Reply:
Yep, that is too long. The python3-ntplib library sets the timeout to 5 seconds, so I have changed it to the same number.